demos/speech_recognition_streaming_demo/python/README.md

```diff
@@ -1,8 +1,8 @@
-# Speech Recognition Python\* Demo
+# Speech Recognition Streaming Python\* Demo
```

Comment:
@AlexeyKruglov I tend to think that we have to change this demo name to mozilla_deepspeech_demo. That will distinguish this demo from already presented at OpenVINO `offline_speech_recognition_demo` and `live_speech_recognition_demo` dedicated for Kaldi models and highlight the fact that demo work with Mozilla deepspeech models only.

@Wovchena having both `offline_speech_recognition_demo` in OpenVINO and `speech_recognition_offline_demo` indicate we have an issue.

@dorloff I think demo which work with Kaldi models only should have `kaldi` modifier in their name

Reply:
Makes sense. For demos this breaks task type based splitting, which isn't that bad to me.

Reply:
Renamed to `speech_recognition_deepspeech_demo`.